corehq/apps/export/views.py

```diff
@@ -1831,11 +1820,10 @@ def get_app_data_drilldown_values(self, in_data):
         try:
             rmi_helper = ApplicationDataRMIHelper(self.domain, self.request.couch_user)
             response = rmi_helper.get_case_rmi_response()
-        except Exception as e:
+        except Exception:
             return format_angular_error(
-                _("Problem getting Create Export Form: {}").format(e.message),
+                _("Problem getting Create Export Form"),
                 log_error=True,
-                exception=e,
```

Comment:
Yeah, so you'd think that removing the exception param would result in not logging the exception. My understanding is that basically passing in the error itself wasn't doing anything. If you look down to https://github.com/dimagi/commcare-hq/pull/16924/files#diff-6071ec21b34b72523fded4adb3cbb516L19, you'll see that `exception` is being passed in _as the message_ to `logging.exception`. I believe the original author just misunderstood `logging.exception`'s function signature. My understanding is that `logging.exception` is just like `logging.error`, but that it includes exc_info (by looking it up from a global place, not by making you pass in an exception object).